lib/src/slider/Slider.test.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { render, fireEvent, act } from "@testing-library/react";
 import DxcSlider from "./Slider";
+import userEvent from "@testing-library/user-event";
 
```

Comment:
Please put this import before the `DxcSlider` import to avoid the warning:

![image](https://user-images.githubusercontent.com/10975289/190179512-2e553173-3431-4285-93cc-df045efbdc9d.png)